admin_app/src/app/content/edit/page.tsx

```diff
@@ -210,28 +261,54 @@ const ContentBox = ({
             } else if (content.content_text === "") {
               setIsContentEmpty(true);
             } else {
-              const handleSaveContent = async (content: Content) => {
-                const content_id = await saveContent(content);
-                if (content_id) {
-                  const actionType = content.content_id ? "edit" : "add";
-                  router.push(
-                    `/content/?content_id=${content_id}&action=${actionType}`,
-                  );
-                }
+              const handleSaveContent = async (content: Content, content_id: number | null) => {
+
+                await saveContent(content, content_id);
+
               };
-              handleSaveContent(content);
+              handleSaveContent(content, content_id);
             }
-          }}
+          }
+          }
         >
           Save
         </Button>
+        <Dialog open={showModal} onClose={() => setShowModal(false)}>
```

Comment:
Added this modal to give the option to add a new language version of a content instead of being redirect to the landing page automatically.